### PR TITLE
[dump] Add ACL modules

### PR DESCRIPTION
**Pull request by stepanblyschak**
Signed-off-by: Stepan Blyschak <stepanb@nvidia.com>

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@vivekreddynv could you please help to review?

File: doc/Dump-Utility.md [hunk 1]
```diff
@@ -782,14 +785,213 @@ req.file = "/etc/sonic/copp_cfg.json"
...
+  "return_values": {"COPP_TRAP|sflow" : {"trap_group" : "queue2_group1"}}
 }
 ```
 
-## 3 **Unit Tests**:
+## 3 **Modules**:
+
+### 3.1 ACL Table
+
+ACL Tables configuration resieds in CONFIG DB in table *ACL_TABLE*. Tables in *ACL_TABLE* may reference another table *ACL_TABLE_TYPE* that holds the table type configuration or may use one of default table types (L3, L3V6, MIRROR, etc.). This information is dumped from CONFIG DB. Orchagent processes *ACL_TABLE* and *ACL_TABLE_TYPE* configuration and manages the following ASIC resources:
```
Comment (vivekreddynv) [inline doc/Dump-Utility.md:799]: typo "resides"

**Review by vivekreddynv — CHANGES_REQUESTED** [review]

File: doc/Dump-Utility.md (hunk 1 above)
Comment (stepanblyschak) [reply doc/Dump-Utility.md]: Thanks, fixed

**Review by vivekreddynv — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@bingwang-ms kindly reminder to review.

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**